content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -234,6 +234,65 @@ the delay value is set too short, the scale of the replicas set may keep thrashi
 usual.
 {{< /note >}}
 
+## Support for resource metrics
+
+Any HPA target can be scaled based on the resource usage of the pods in the scaling target.
+When definining the pod specification the resource requests like `cpu` and `memory` should
+be specified. This is used to determine the resource utilization and used by the HPA controller
+to scale the target up or down. To use resource utilzation based scaling specify a metric source
+like this:
+
+```yaml
+type: Resource
+resource:
+  name: cpu
+  target:
+    type: Utilization
+    averageUtilization: 60
+```
+With this metric the HPA controller will keep the average utilization of the pods in the scaling
+target at 60%. Utilization is the ratio between the current usage of resource to the requested
```

Comment:
This is a bit confusing.

- Does this mean the average CPU utilization of all Pods has to be exactly 60%, or else the HPA will start scaling the target?
- Does this mean the average CPU utilization of all containers in a single Pod rather than the average across all Pods?

Reply:
There is a tolerance factor which is passed as a flag to the controller. So yes, the utilization has to be ~60% and the HPA controller scales the replicas to keep in within that range. I've updated the docs with a link to the algorithm details.